### PR TITLE
Add `dev` build of Package

### PR DESCRIPTION
**Pull request by michaelweinold**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

I tried to follow [CFEP-05](https://github.com/conda-forge/cfep/commit/c08033304fd31833c17f1869850d09c5da3f81b3) - hopefully I didn't forget anything 🤞🏻

### CODE REVIEW

**Comment by michaelweinold** [comment]
This would close https://github.com/brightway-lca/brightway-hub/issues/19

File: recipe/conda_build_config.yaml
Comment (m-rossi) [inline recipe/conda_build_config.yaml:1]: Just a minor thing: Add an empty line at the end.

**Comment by m-rossi** [comment]
Furthermore it seems the requirements are out of date for the dev version

**Comment by michaelweinold** [comment]
Hmm... I'm seeing [a test failure](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=765239&view=logs&j=656edd35-690f-5c53-9ba3-09c10d0bea97&t=986b1512-c876-5f92-0d81-ba851554a0a3&l=636):

> `TESTS FAILED: bw2data-4.0.dev21-pyh526c96b_0.conda`

I just updated the requirements according to match [the setup.py of the latest dev version](https://github.com/brightway-lca/brightway2-data/blob/4.0.DEV21/setup.py). Let's hope this resolves the issue.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -64,3 +59,4 @@ extra:
     - michaelweinold
     - cmutel
     - m-rossi
+    - michaelweinold
```
Comment (m-rossi) [inline recipe/meta.yaml]: That's a duplicate.

**Comment by m-rossi** [comment]
@conda-forge-admin, please rerender

**Review by m-rossi — APPROVED** [review]

**Pull request merged**